src/engine/Sort.cpp

```diff
@@ -77,6 +77,7 @@ void Sort::computeResult(ResultTable* result) {
   result->_resultTypes.insert(result->_resultTypes.end(),
                               subRes->_resultTypes.begin(),
                               subRes->_resultTypes.end());
+  // TODO: Shouldn't we use std::move here?
```

Comment:
It's a `shared_ptr`, and `move` would delete the local vocab from the cached subresult which could lead to strange bugs (either empty results or segfaults). But we could discuss and comment this overall.

Reply:
You are absolutely right